.circleci/config.yml

```
@@ -442,17 +453,17 @@ jobs:
               cd habitat-website
               git submodule update --init
 
-              for dir in habitat-sim habitat-cpp
+              for dir in habitat-sim habitat-cpp habitat-api
               do
                   rm -rf published/docs/${dir}
                   cp -r ../habitat-sim/build/docs-public/${dir} published/docs/.
```

```suggestion
                  cp -r ../${dir}/build/docs-public/${dir} published/docs/.
```
Even more complicated as API docs are in `./habitat-api/docs/build/docs-public` as I understand.